compose/mpp/demo/src/jsMain/resources/index.html

```diff
@@ -4,7 +4,9 @@
     <meta charset="UTF-8">
     <title>compose multiplatform web demo</title>
     <script src="skiko.js"> </script>
-    <link type="text/css" rel="stylesheet" href="styles.css">
+    <meta name="viewport"
```

Comment:
The same comment as with viewport. if this has nothing to do with touch events - let's keep it out of scope of this particular commit - however small (and justified) the changes are.